src/deprecated.jl

```diff
@@ -439,3 +439,10 @@ import Base: map
 @deprecate map(f::Base.Callable, gd::GroupedDataFrame) combine(f, gd, ungroup=false)
 @deprecate map(f::Pair{<:ColumnIndex}, gd::GroupedDataFrame) combine(f, gd, ungroup=false)
 @deprecate map(f::Pair, gd::GroupedDataFrame) combine(AsTable(first(f)) => last(f), gd, ungroup=false)
+
+@deprecate DataFrame!(args...; kwargs...) begin
+    if :copycols in keys(kwargs)
```

Comment:
Isn't this going to be printed too? I think you need to call `depwarn` manually. Or just drop this check, people will get an error anyway.

Reply:
it is going to be printed. If we remove it then we change the behavior of `DataFrame!` in some corner cases. The issue is that when someone writes `DataFrame!(..., copycols=false)` with `depwarn` you will get:
1. a warning to use `copycols=false`
2. an error that you are not allowed to pass `copycols=false`

But now as I think of it - I can put `depwarn` after the error and it will be OK. I will change it

Reply:
I the end I decided to skip the error part as it threw error earlier in order to bet a cleaner deprecation.